dvc/repo/__init__.py

```diff
@@ -418,10 +418,15 @@ def stages(self):
         return get_stages(self.graph)
 
     def find_outs_by_path(self, path, outs=None, recursive=False):
+        abs_path = (
+            os.path.join(self.root_dir, path)
+            if not os.path.isabs(path)
+            else path
+        )
```

Comment:
It used to support paths relative to curdir, it won't after this change. Now it understands non absolute paths as relative to root dir. So this is a change in semantics. Will it break something? Why do we need it?

Reply:
Oh, you are right, should make abspath out of it. 
I moved abspath calculation here so that I could pass `path` to OutputNotFoundError. In case of relative path passed by user, OutputNotFoundError can use the input for error message, which is more meaningful than printing full path or path relative to curdir, as it used to be.
https://asciinema.org/a/280576
vs
https://asciinema.org/a/280577

Reply:
But it should stay relative to curdir for outs within current repo. Like it or not, but it is what we are doing everywhere now.